### PR TITLE
Merge Network 2018 05 01 to preview

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by EvgenyAgafonchikov** [comment]
The PR drops a pair of VPN Gateway functions cause they removed from SDK. See [this PR](https://github.com/Azure/azure-rest-api-specs/pull/2975) for details

File: tools/AzureRM/AzureRM.psm1 [hunk 1]
```diff
@@ -1 +1 @@
-# Placeholder for script template
+# Placeholder for script template
```
Comment (MikhailTryakhov) [inline tools/AzureRM/AzureRM.psm1]: This is a gitub issues. This file hasn't been changed

**Comment by praries880** [comment]
@MikhailTryakhov dod you mean to target the branch [2018-05-01](https://github.com/Azure/azure-powershell/tree/Network-2018-05-01) instead of preview?

**Comment by maddieclayton** [comment]
Closing per discussion today.  Feel free to reopen when all the SDK changes have been made and published.

**Pull request closed without merging**